__tests__/components/Attachment.test.tsx

```diff
@@ -95,3 +95,135 @@ describe("Attachment Component", () => {
     ).not.toBeInTheDocument();
   });
 });
+
+describe("MIME-type icon rendering", () => {
```

Comment:
If we move to an external mapping list we can write this test using it.each see `__tests__/lib/planningApplication/search.test.ts`